api/src/main/java/org/apache/apex/api/ApexPlugin.java

```diff
@@ -18,12 +18,15 @@
  */
 package org.apache.apex.api;
 
+import org.apache.hadoop.classification.InterfaceStability;
+
 import com.datatorrent.api.Component;
 
 /**
  * Marker interface for ApexPlugins.
  * @param <T>
  */
+@InterfaceStability.Evolving
 public interface ApexPlugin<T extends ApexPluginContext> extends Component<T>
```

Comment:
Yes, can it be in a separate subpackage like plugins? Also if there are two different plugins, can their names reflect the difference. Apex prefix is too generic and does not do anything other than makes name longer.